content/en/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,43 @@
+
+PID Limiting features allow administrators are able to provide pod-to-pod PID isolation by defaulting the number of PIDs per pod. In addition, administrators can enable node-to-pod PID isolation by reserving a number of allocatable PIDs to user pods via node allocatable.
+
+<!-- body -->
+
+Process IDs (PIDs) are a fundamental resource on Linux hosts. It is trivial to hit the task limit without hitting any other resource limits and cause instability to a host machine. Administrators require mechanisms to ensure that user pods cannot induce PID exhaustion that prevents host daemons (runtime, kubelet, etc) from running. In addition, it is important to ensure that PIDs are limited among pods in order to ensure they have limited impact to other workloads on the node.
+
+You can configure a kubelet to limit the number of PIDs a given pod can consume. If that machine supports `32,768` PIDs and `100` pods, one can give each pod a budget of `300` PIDs to prevent total exhaustion of PIDs. If the admin wants to overcommit PIDs similar to cpu or memory, they may do so as well with some additional risks. Either way, no one pod can bring the whole machine down. This will generally prevent against simple fork bombs from taking over your cluster.
+
+Per-pod PID limiting allows administrators to protect one pod from another, but does not ensure if all pods on the machine can protect the node, and the node agents themselves from falling over.
+
+Node PID limiting provides isolation of PIDs from end user workloads on a pod from the node agents (kubelet, runtime, etc.). The admin is able to reserve a specific number of PIDs - similar to how one reserves CPU or memory today - and ensure they are never consumed by pods on that machine.
+
+PID limiting is a an important part of the family of [compute resources](/docs/concepts/configuration/manage-resources-containers/) requests and limits.
+
+## Enabling Node PID limits
+
+In order to enable the support to limiting PIDs on the Node, use the parameter `pid=<number>` in the `--system-reserved` and `--kube-reserved` options. The value is specified to ensure that the specified number of process IDs will be reserved for the system as a whole and for Kubernetes system daemons respectively.
+
+**Note:** Before the version 1.20, these feature require feature flags [`SupportNodePidsLimit`](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates:~:text=SupportNodePidsLimit) to work.
+
+## Enabling Pod PID limits
+
+Pod PID limit allows to limit the number of processes running in a pod. Cluster administrators can use command line parameter [`--pod-max-pids`](/docs/reference/command-line-tools-reference/kubelet/#options:~:text=%2D%2Dpod%2Dmax%2Dpids%20int) or [kubelet config file](/docs/tasks/administer-cluster/kubelet-config-file/) setting [`PodPidsLimit`](https://github.com/kubernetes/kubernetes/blob/f5a54e3f5862994f8cf422a60f4973583e4f98ba/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L484-L490).
```

Comment:
```suggestion
Kubernetes allows you to limit the number of processes running in a Pod. You specify this limit at the node level, rather than configuring it as a resource limit for a particular Pod. Each Node can have a different PID limit.  
To configure the limit, you can specify the command line parameter `--pod-max-pids` to the kubelet, or set `PodPidsLimit` in the kubelet [configuration file](/docs/tasks/administer-cluster/kubelet-config-file/).
```